src/file-upload/internal.tsx

```diff
@@ -79,10 +90,7 @@ function InternalFileUpload(
   const onFileRemove = (removeFileIndex: number) => {
     const newValue = value.filter((_, fileIndex) => fileIndex !== removeFileIndex);
     fireNonCancelableEvent(onChange, { value: newValue });
-    setRemovedFileIndex(removeFileIndex);
-    if (value.length === 1) {
-      fileInputRef.current?.focus();
```

Comment:
No need to trigger the focusing here because it is handled by useListFocusController with `outsideSelector`.